bundle/regal/config/config.rego

```diff
@@ -14,7 +14,27 @@ docs["resolve_url"](url, category) := replace(
 
 user_config := data.regal_user_config
 
-merged_config := object.union(data.regal.config.provided, user_config)
+config_union := object.union(data.regal.config.provided, user_config)
+
+merged_config["ignore"] := config_union.ignore
+
+merged_config["rules"] := merged_rules
+
+merged_rules[category] := rules if {
+	some category, _rules in config_union.rules
+	rules := {rule_name: empty_level_to_provided(category, rule_name, rule_conf) |
+		some rule_name, rule_conf in _rules
+	}
+}
+
+empty_level_to_provided(_, _, conf) := conf if conf.level != ""
+
+empty_level_to_provided(category, title, conf) := object.union(
```

Comment:
I spent a little while thinking about this, I wonder if an explanatory comment might be handy for the future since the level is being processed as a special case.

Reply:
Nah, I'm just having a panic at attack at how slow I am to read Rego, still! 😅